src/modules/components/asset/assetTransfer/assetTransfer.tsx

```diff
@@ -36,7 +36,7 @@ export interface IAssetTransferProps extends IWeb3ComponentProps {
     /**
      * Transaction hash.
      */
-    hash: string;
+    hash?: string;
```

Comment:
As the tickets says that we should have a link to the token address, shouldn't we remove this `hash` property and build the asset link instead?